syncd/FlexCounter.cpp

```diff
@@ -31,6 +31,13 @@ static const std::string ATTR_TYPE_PG = "Priority Group Attribute";
 static const std::string ATTR_TYPE_MACSEC_SA = "MACSEC SA Attribute";
 static const std::string ATTR_TYPE_ACL_COUNTER = "ACL Counter Attribute";
 
+// --> Needed for agg voq stats
+static sai_queue_api_t *sai_queue_api = NULL;
+static swss::DBConnector configDb("CONFIG_DB", 0);
+static swss::Table cfgDeviceMetaDataTable(&configDb, CFG_DEVICE_METADATA_TABLE_NAME);
+static swss::DBConnector counterDb("COUNTERS_DB", 0);
+// <-- agg voq stats end
+
```

Comment:
please remove all global variables, all connectsion for db should come from parametrs to the class not had coded

Reply:
Fixed.